Magento_Theme/templates/html/pager.phtml

```diff
@@ -52,12 +52,12 @@
                 >
                     <?php foreach ($block->getAvailableLimit() as $_key => $_limit): ?>
                         <option
-                            value="<?= $block->getLimitUrl($_key) ?>"
+                            value="<?= $block->escapeUrl($block->getLimitUrl($_key)) ?>"
```

Comment:
Magento2 core uses here escapeHtmlAttr but I think we should use escapeUrl here as you did so let's leave it as you did.